packages/bpk-component-loading-button/src/BpkLoadingButton.js

```diff
@@ -34,7 +34,8 @@ const getPropsIcon = props => {
 
   if (loading) {
     return iconLoading;
-  } else if (disabled) {
+  }
+  if (disabled) {
```

Comment:
Isn't this technically less performant because now this check will fire every time?

Reply:
Nah because line 36 has a return statement. here is the eslint docs https://eslint.org/docs/rules/no-else-return

Reply:
Ah so it does!